src/Common/src/System/Net/Logging/GlobalLog.cs

```diff
@@ -173,41 +183,52 @@ public static void Enter(string functionName, string parameters)
             EventSourceLogging.Log.FunctionStart(functionName, parameters);
         }
 
-        public static void Assert(bool condition, string messageFormat, params object[] data)
+        public static void Assert(bool condition, string message)
```

Comment:
Given the changes, I see this method as promoting bad patterns. 
If it can't be removed right away, please add a TODO to ensure that all modules using GlobalLog are modified as well.